linera-core/src/client.rs

```diff
@@ -203,14 +211,40 @@ pub enum MessagePolicy {
 }
 
 impl MessagePolicy {
+    pub fn new(
+        blanket: BlanketMessagePolicy,
+        restrict_chain_ids_to: Option<HashSet<ChainId>>,
+    ) -> Self {
+        Self {
+            blanket,
+            restrict_chain_ids_to,
+        }
+    }
+
+    #[tracing::instrument(level = "trace", skip(self))]
+    fn accepts(&self, bundle: &mut IncomingBundle) -> bool {
```

Comment:
Maybe `handle`, or `apply`?
If I'm reading this correctly, `policy.accepts(&mut bundle)` returns `true` if it changes the bundle to `Reject` (i.e. doesn't _accept_ it) but needs to include it in the block?

Reply:
Yes I think `handle` is better in this case.